network-api/networkapi/wagtailpages/models.py

```diff
@@ -310,6 +310,14 @@ def __str__(self):
         return self.page.title + "->" + self.highlight.title
 
 
+class PrimaryPage(Page):
+    body = StreamField(base_fields)
+
+    content_panels = Page.content_panels + [
+        StreamFieldPanel('body'),
+    ]
+
+
```

Comment:
let's also add a `parent_page_types = ['Homepage']` so that people can't spuriously add these under other page types